### DISCUSSION TITLE
enh: Configurable default theme

### DISCUSSION THREAD

**Discussion by alexanderahrens** [open-webui/open-webui#6277] [Ideas]
# Offered as Enterprise feature.

**Comment by justinh-rahb** [comment]
AFAIK there's already an effort in the plans for making an embeddable javascript widget, perhaps this may solve your requirement better than an iFrame would anyway. This should be fully themeable.

**Comment by alexanderahrens** [comment]
Hello @justinh-rahb, that sounds promising. I'd be happy to test the javascript widget once available!

**Comment by Genai-labs** [comment]
Hello, this feature seems interesting, do you have any update about it ?
